components/byu-search/byu-search.js

```diff
@@ -0,0 +1,27 @@
+import { html, css, customElement, LitElement, unsafeCSS } from 'lit-element'
+import style from './byu-search.sass'
+
+@customElement('byu-search')
+export class BYUSearch extends LitElement {
+
+  static get styles () {
+    return css`${unsafeCSS(style)}`
+  }
+
+  // TODO: Implement BYU Search features
+
+  render () {
+    return html`
+<form class="byu-search" action="/">
+    <label class="byu-search-label" for="site-search">Search</label>
+    <input type="search" id="byu-site-search" name="q" aria-label="Site search" placeholder="Search">
+    <button class="byu-search-btn">
```

Comment:
I ran into one weird issue with the search - once you start typing you get an 'X' over top of the magnifying glass icon:

![image](https://user-images.githubusercontent.com/19493582/61137157-3eee0200-a482-11e9-8714-42bdc197eb34.png)

Reply:
This pull request only focuses on the display of the components, not on functionality. I would consider this a functionality issue because it only happens when the component is used and therefore out of the scope of this PR.